index.html

```diff
@@ -13,11 +13,30 @@
     <link rel="stylesheet" href="styles/main.css">
     <link rel="stylesheet" href="styles/footer.css">
   </head>
-  <body>
-    <div class="div--maincolor">
-      <h1 class="h1">My First Webpage</h1>
-      <p class="p">Hello World!</p>
-      <img src="img/header/logo_prtimes.svg" alt="PR TIMES"/>
-    </div>
+  <body class="container">
+    <header class="container__header header">
+      <div class="header-content header__header-content">
+        <div class="title header-content__title">
+          <img class="title__logo" src="img/header/logo_prtimes.svg" alt="PR TIMES"/>
+          <h1 class="title__text">プレスリリース・ニュースリリース配信サービスのPR TIMES</h1>
```

Comment:
ロゴの部分は一般だと、リンクになるので、以下のように
```suggestion
         <a href="/" title="ホームページに戻る">
          <img class="title__logo" src="img/header/logo_prtimes.svg" alt="PR TIMES"/>
          <h1 class="title__text">プレスリリース・ニュースリリース配信サービスのPR TIMES</h1>
         </a>
```
しても良いと思います。

Reply:
[こちらのcommit](https://github.com/ShunDeveloper/prtimes_htmlcss/pull/10/commits/d8d8c54f8b3e403910371e0eaef328784c438a5d)で修正しました.